pages/api/user/[id].ts

```diff
@@ -25,8 +25,22 @@ export default async function handle(
         }
         ResponseUtil.returnOK(res, user);
         return;
+    } else if (req.method == "POST") {
+        // Obtain the data of the user to be created
+        const newUserData = req.query;
+        const newUser = createUser(newUserData);
+
+        if (!newUser) {
+            ResponseUtil.returnBadRequest(
+                res,
+                "User could not be created. Please verify that you have entered the information correctly.",
+            );
```

Comment:
If a user can not be created in our database - that probably means there's an internal error. We should verify that their request is good before storing it in our database and if it isn't -then we can return a bad request.